map/create-and-save-map/README.md

```diff
@@ -16,25 +16,20 @@ To create and save a map to your portal for use in an app:
 
 1. Create an `ArcGISMap` with a `Basemap` and operational layers
 2. Create an `OAuthConfiguration` with your portal and app credential.
-3. Add the configuration to the `AuthenticationMangager`
-4. Create a `Portal` and load it. Use a custom `AuthenticationChallengeHandler` to
-  authenticate with your username and password
-5. Once authenticated, save the map by calling `map.saveMapAsAsync()`, passing in the title, tags,
-  description, and portal folder
+3. Add the configuration to the `AuthenticationManager`
+4. Create a `Portal` and load it. Use a `DefaultAuthenticationChallengeHandler` to authenticate with your username and password
+5. Once authenticated, save the map by calling `map.saveMapAsAsync()`, passing in the title, tags, description, and portal folder
 
 ## Relevant API
 
-* AuthenticationChallengeHandler
 * ArcGISMap
 * ArcGISMapImageLayer
+* AuthenticationManager
 * Basemap
+* DefaultAuthenticationChallengeHandler
 * MapView
```

Comment:
Remove `ArcGISMap`, `ArcGISMapImageLayer`, `Basemap`, and `MapView`  as they are not directly related to the purpose of the sample.

Reply:
Good spot on this, I've also updated the readme to have the additional sections that are considered mandatory (use case and tags)